src/etwlogger_driver.c

```diff
@@ -9,9 +9,16 @@
 #include "windows.h"
 
 #include "azure_c_shared_utility/xlogging.h"
-#include "azure_c_shared_utility/etwlogger.h"
+
 #include "azure_c_shared_utility/consolelogger.h"
 
+#ifdef USE_TRACELOGGING
+#include "TraceLoggingProvider.h"
+#include "evntrace.h"
+#else
+#include "azure_c_shared_utility/etwlogger.h"
+#endif
```

Comment:
Like said earlier, let's have just the tracelogging flavor.
And I shall not say it anymore now :_).

Reply:
Agree. Will be done with a later task.

---
In reply to: [337707663](https://github.com/Azure/azure-c-shared-utility/pull/401#discussion_r337707663) [](ancestors = 337707663)